src/App.jsx

```diff
@@ -1,51 +1,116 @@
+import Login from './components/Authentication/Login';
+import Logout from './components/Authentication/Logout';
+import ProtectedRoute from './common/ProtectedRoute';
+import EmailAction from './components/Authentication/EmailAction';
+
+import CommonTableExample from './pages/CommonTableExample';
+import AdminPage from '../src/pages/AdminPage';
 import HomePage from './pages/HomePage';
+import AccountPage from './pages/AccountPage';
 import SectionPage from './pages/SectionPage';
 import MonitorLogPage from './pages/MonitorLogPage';
-import AccountPage from './pages/AccountPage';
-import AdminPage from './pages/AdminPage'; // monitor-log table
 import EndangeredSpeciesPopup from './pages/EndangeredSpeciesPopup';
+import Register from './components/Authentication/Register';
+// import NewUser from './components/Authentication/NewUser';
 import PeoplePage from './pages/PeoplePage';
 import Species from './pages/Species';
 import Navbar from './components/Navbar/Navbar';
-import theme from './theme/theme';
 import Footer from './components/Footer/Footer';
+import AdminInviteModal from './components/Authentication/AdminInviteModal';
 
-import CommonTableExample from './pages/CommonTableExample';
+import theme from './theme/theme';
+import AUTH_ROLES from './common/auth_config';
+
+const { SUPER_ADMIN_ROLE, ADMIN_ROLE, VOLUNTEER_ROLE } = AUTH_ROLES.AUTH_ROLES;
 
 function App() {
   return (
     <ChakraProvider theme={theme}>
-      <Router>
-        <Box className="page-container">
-          <Box className="content-wrap">
-            <Navbar isAdmin />
-            <Routes>
-              {/* Add routes as needed; route names subject to change */}
-              <Route exact path="/" element={<HomePage />} />
-              <Route exact path="/account" element={<AccountPage />} />
-              <Route exact path="/create-log" element={<MonitorLogPage />} />
-              <Route exact path="/sections" element={<SectionPage />} />
-              <Route
-                exact
-                path="/endangered"
-                element={<EndangeredSpeciesPopup adultName="Snowy Plovers" />}
-              />
-              <Route exact path="/species" element={<Species />} />
-              {/* Admin only routes (TO DO, make admin only) */}
-              <Route exact path="/people" element={<PeoplePage />} />
-              <Route exact path="/contacts" />
-              <Route exact path="/map" />
-              <Route exact path="/logs" element={<AdminPage />} />
-              <Route exact path="/common-table-example" element={<CommonTableExample />} />
-            </Routes>
+      <CookiesProvider>
+        <Router>
+          <Box className="page-container">
+            <Box className="content-wrap">
+              <Navbar isAdmin />
+              <Routes>
+                {/* Add routes as needed; route names subject to change */}
+                <Route path="/invite-user/:inviteID" element={<InviteLandingPage />} />
```

Comment:
Can we call this route `/register` and delete the register route on line 102, since I don't thing that's being used? I think it makes more sense.

When you change it please reflect it wherever it's used throughout the app (like in emails and stuff)